src/test/java/com/epam/tc/hw5/cucumber/hook/CucumberHooks.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.tc.hw5.cucumber.hook;
+
+import com.epam.tc.hw5.cucumber.context.TestContext;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.BeforeAll;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class CucumberHooks {
+
+    private WebDriver driver;
+
+    @BeforeAll
+    public static void setUpAll() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @Before
+    public void setUp() {
+        driver = new ChromeDriver();
```

Comment:
Please set up driver window to maximized size as it's one of requirements

Reply:
Window maximized